sql/sql_i_s.h

```diff
@@ -142,8 +142,10 @@ class Varchar: public Type
 public:
   Varchar(uint length) :Type(&type_handler_varchar, length, false)
   {
+    // utf8mb3 Varchars longer than MAX_FIELD_VARCHARLENGTH/3 become Longtexts
     DBUG_ASSERT(length * 3 <= MAX_FIELD_VARCHARLENGTH);
   }
+  Varchar(): Type(&type_handler_varchar, MAX_FIELD_VARCHARLENGTH/3, false) {}
```

Comment:
https://mariadb.com/kb/en/unicode/ says 11.8 wants to change `utf8` to `utf8mb4`.
Assuming the default charset is simply ‘utf8’, I’ll let that work change these `3`s to `4`s, in hopes that it notices it creates unexpected https://github.com/MariaDB/server/pull/3772#discussion_r1919479374 changes too.